src/main/java/org/sonarlint/intellij/config/project/ProjectExclusionsPanel.java

```diff
@@ -56,7 +56,7 @@ public ProjectExclusionsPanel(Project project) {
         + "\">override</a> your locally defined exclusions.";
       editorPane2.setVisible(false);
     } else {
-      message = "When a project is connected to SonarQube or SonarCloud, exclusions defined in the server's General Settings " +
```

Comment:
We should use `SonarQube (Server, Cloud)` as stated [here](https://zeroheight.com/3bcff7bd1/p/84823f-product-descriptions/b/368dfe)